en/services/parameter.md

```diff
@@ -176,18 +185,19 @@ The sequence of operations is:
      All targeted components should respond with parameters (or ignore the request if they have none).
    - The GCS is expected to accumulate parameters from all responding systems.
    - The timeout/retry behaviour is GSC dependent.
-1. The targeted component(s) should respond, sending all parameters individually in [PARAM_VALUE](../messages/common.md#PARAM_VALUE) messages.
+2. The targeted component(s) should respond, sending all parameters individually in [PARAM_VALUE](../messages/common.md#PARAM_VALUE) messages.
    - Allow breaks between each message in order to avoid saturating the link.
    - Components with no parameters should ignore the request.
-1. GCS starts timeout after each `PARAM_VALUE` message in order to detect when parameters are no longer being sent (that the operation has completed).
+3. GCS also starts timeout after each `PARAM_VALUE` message in order to detect when parameters are no longer being sent (that the operation has completed).
+4. If the targeted component does not have any parameters then it should ignore the `PARAM_REQUEST_LIST` request.
+   The GCS may resend the request on timeout if no `PARAM_VALUE` is received, but should cease after resends.
```

Comment:
Should we create an error for this case so that no need for a retry on PARAM_REQUEST_LIST?